### PR TITLE
feat: aries-framework-go skeleton slideshow

### PR DESCRIPTION
**Pull request by troyronda**
Signed-off-by: Troy Ronda <troy.ronda@securekey.com>

### CODE REVIEW

File: 2020/01/aries-framework-go/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# aries-framework-go
```
Comment (llorllale) [inline 2020/01/aries-framework-go/README.md]: Please add a couple of lines describing what this repo is for.

File: 2020/01/aries-framework-go/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "aries-framework-go",
```
Comment (llorllale) [inline 2020/01/aries-framework-go/package.json]: Name needs to be scoped.

File: 2020/01/aries-framework-go/README.md (hunk 1 above)
Comment (troyronda) [reply 2020/01/aries-framework-go/README.md]: done

File: 2020/01/aries-framework-go/package.json (hunk 2 above)
Comment (troyronda) [reply 2020/01/aries-framework-go/package.json]: done (although I'm not sure if it makes sense as a published package anyways).

Comment (llorllale) [reply 2020/01/aries-framework-go/package.json]: ok - understandable now that I see what the repo is for

**Pull request merged**